src/operators/concat-static.ts

```diff
@@ -15,7 +15,6 @@ export default function concat<R>(...observables: Array<Observable<any> | Schedu
   let args = <any[]>observables;
   if (typeof (args[observables.length - 1]).schedule === 'function') {
     scheduler = args.pop();
-    args.push(1, scheduler);
```

Comment:
It's because the lines below it were an oversight... I was building arguments to pass to `mergeStatic.apply`, then it looks like I fell asleep in my chair and forgot to do the rest.

Below, we should _not_ be calling `.mergeAll` on the observable itself, because due to modularity, it might not be there, rather, we should calling the static `merge` function with apply, which we should be importing

Something like:

```js
return mergeStatic.apply(this, args);
```

*THAT* is why I was building and maintaining args.